dockstore-client/src/main/java/io/dockstore/client/cli/Client.java

```diff
@@ -215,6 +217,47 @@ public static String getCurrentVersion(String installLocation) {
     }
 
     /**
+     * Check if the date of the current is later or earlier than latest version
+     * @param current
+     * @param latest
+     * @return
+     */
+    public static Boolean compareVersion(String current, String latest){
+        URL urlCurrent, urlLatest;
+        try{
+            urlCurrent = new URL("https://api.github.com/repos/ga4gh/dockstore/releases/tags/"+current);  //for testing, change to current later on
+            urlLatest = new URL("https://api.github.com/repos/ga4gh/dockstore/releases/tags/"+latest);
+
+            ObjectMapper mapper = new ObjectMapper();
+            Map<String, Object> mapCur, mapLat;
+            int idCur,idLat;
+            String prerelease;
+
+            try {
+                //id of latest version
+                mapLat = mapper.readValue(urlLatest, Map.class);
+                idLat = Integer.parseInt(mapLat.get("id").toString());
+                //id and prerelease of current version
+                mapCur = mapper.readValue(urlCurrent, Map.class);
+                idCur = Integer.parseInt(mapCur.get("id").toString());
+                prerelease = mapCur.get("prerelease").toString();
+
+                //check if currentVersion is earlier than latestVersion or not
+                //id will be bigger if newer, prerelease=true if unstable
+                //newer return true, older return false
+                if(prerelease.equals("true") && (idCur>idLat)) {
+                    return true;  //current is newer and not stable
+                }
+            } catch (IOException e) {
+                // e.printStackTrace();
```

Comment:
For these two exceptions, you should probably fail-fast rather than fall through and always return false. 
Please log an error and then throw a new RuntimeException to shut down the program